file_stream/file_stream_test.go

```diff
@@ -0,0 +1,101 @@
+package file_stream
+
+import (
+	"context"
+	"io"
+	"os"
+	"testing"
+	"time"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestFileStream_MoreBytes(t *testing.T) {
+	ctx := context.Background()
+
+	Convey("file_stream.MoreBytes()", t, func() {
+		file_name := "/tmp/tail_test.log"
```

Comment:
Tests should clean up after themselves - it looks like this will leave a file behind each time it runs.